homeassistant/components/verisure/binary_sensor.py

```diff
@@ -66,3 +71,32 @@ def available(self):
     def update(self):
         """Update the state of the sensor."""
         hub.update_overview()
+
+
+class VerisureEthernetStatus(BinarySensorDevice):
+    """Representation of a Verisure VBOX internet status."""
+
+    @property
+    def name(self):
+        """Return the name of the binary sensor."""
+        return "Verisure Ethernet status"
+
+    @property
+    def is_on(self):
+        """Return the state of the sensor."""
+        return hub.get_first("$.ethernetConnectedNow")
```

Comment:
It would be good if the library would be rewritten to not require the user to know the protocol details, and instead expose different methods for different features of the hub. We try to avoid this level of protocol details inside home assistant.

But this is out of scope of this PR.